test/Unit/Durable/DurableTimerTests.cs

```diff
@@ -0,0 +1,187 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Test.Durable
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using Microsoft.Azure.Functions.PowerShellWorker.Durable;
+    using Moq;
+    using Xunit;
+
+    public class DurableTimerTests
+    {
+        private const int LongIntervalSeconds = 5;
+        private const int ShortIntervalSeconds = 3;
+        private static readonly DateTime startTime = DateTime.UtcNow;
+        private readonly DateTime fireAt = startTime.AddSeconds(ShortIntervalSeconds);
+        private static readonly DateTime restartTime = startTime.AddSeconds(LongIntervalSeconds);
+        private static readonly DateTime shouldNotHitTime = restartTime.AddSeconds(LongIntervalSeconds);
+        private OrchestrationInvoker _orchestrationInvoker = new OrchestrationInvoker();
+        private OrchestrationBindingInfo _orchestrationBindingInfo = new OrchestrationBindingInfo(
+                                                                            "ContextParameterName",
+                                                                            new OrchestrationContext());
+        private Mock<IPowerShellServices> _mockPowerShellServices = new Mock<IPowerShellServices>();
```

Comment:
Not used?

Reply:
Oops it looks like I missed these after I removed the unnecessary methods.